src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -831,10 +558,10 @@ public void CompleteTrain()
                 _ldaTrainer.Train(""); /* Need to pass in an empty string */
             }
 
-            public void Output(in VBuffer<Double> src, ref VBuffer<Float> dst, int numBurninIter, bool reset)
+            public void Output(in VBuffer<Double> src, ref VBuffer<float> dst, int numBurninIter, bool reset)
             {
                 // Prediction for a single document.
-                // LdaSingleBox.InitializeBeforeTest() is NOT thread-safe.
+                // LdafloatBox.InitializeBeforeTest() is NOT thread-safe.
```

Comment:
>float [](start = 22, length = 5)

revert

Reply:
fixed

---
In reply to: [233583598](https://github.com/dotnet/machinelearning/pull/1410#discussion_r233583598) [](ancestors = 233583598)